injector/src/main/java/edu/ucr/cs/riple/injector/location/OnField.java

```diff
@@ -58,8 +62,8 @@ protected void fillJsonInformation(JSONObject res) {
   }
 
   @Override
-  protected boolean applyToMember(NodeList<BodyDeclaration<?>> clazz, Change change) {
-    final boolean[] success = {false};
+  protected Modification applyToMember(NodeList<BodyDeclaration<?>> clazz, Change change) {
+    final Modification[] modification = new Modification[1];
```

Comment:
Is this an array just to make it mutable?  If so, I think a more common pattern is to use `AtomicReference` for this purpose

Reply:
Yes, exactly. Thank you for the suggestion, changed it 15586f197edd93b52343e635c4386601abe1ee5b.